src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -1017,16 +1017,24 @@ void SmartScript::ProcessAction(SmartScriptHolder& e, Unit* unit, uint32 var0, u
 
             for (ObjectList::const_iterator itr = targets->begin(); itr != targets->end(); ++itr)
             {
-                if (!IsCreature(*itr))
+                if (!IsCreature(*itr) || !IsGameObject(*itr))
```

Comment:
This line doesn't exist in the original diff: https://github.com/TrinityCore/TrinityCore/issues/15020#issuecomment-118949343